lib/core/src/Cardano/Wallet/DB/Sqlite.hs

```diff
@@ -491,6 +463,30 @@ migrateManually tr defaultFieldValues =
                 | otherwise                       -> ColumnMissing
             _ -> TableMissing
 
+    -- | A migration for adding a non-existing column to a table. Factor out as
+    -- it's a common use-case.
+    addColumn
+        :: Sqlite.Connection
+        -> DBField
+        -> Text
+        -> IO ()
+    addColumn conn field value = do
+        isFieldPresent conn field >>= \case
+            TableMissing ->
+                traceWith tr $ MsgManualMigrationNotNeeded field
+            ColumnMissing -> do
+                traceWith tr $ MsgManualMigrationNeeded field value
+                query <- Sqlite.prepare conn $ T.unwords
+                    [ "ALTER TABLE", tableName field
+                    , "ADD COLUMN", fieldName field
+                    , fieldType field, "NOT NULL", "DEFAULT", value
+                    , ";"
+                    ]
+                _ <- Sqlite.step query
+                Sqlite.finalize query
+            ColumnPresent ->
+                traceWith tr $ MsgManualMigrationNotNeeded field
```

Comment:
I originally started adding the withdrawal to the `TxMeta` table but I reverted and went for a new table instead since it behaves much more like current TxIn and TxOut. I however kept this refactor as I thought it wouldn't hurt :blush: